### PR TITLE
[ADH Space] Hotfixes

### PR DESCRIPTION
**Pull request by enzozccr**

### CODE REVIEW

File: front/style/manager-space/_tables.scss [hunk 1]
```diff
@@ -292,3 +292,17 @@
         }
     }
 }
+
+td.municipal-candidate-tags {
```
Comment (ottaviano) [inline front/style/manager-space/_tables.scss]: `tbody tr td.municipal-candidate-tags span` will not be enough as the selector for avoiding add a new class to each span?

Comment (enzozccr) [reply front/style/manager-space/_tables.scss]: hmm, yes it might be enough !

Comment (enzozccr) [reply front/style/manager-space/_tables.scss]: up, https://github.com/EnMarche/en-marche.fr/pull/3815/commits/9f771eb1e58a98169fd4479fd100b6fa1c872fc6

Comment (ottaviano) [reply front/style/manager-space/_tables.scss]: nice, thanks! already approved ;)

**Pull request merged**